### PR TITLE
GAME-14 - Create new Express.js route /game with blank layout

### PR DESCRIPTION
**Pull request by Birek100**
Add route to game.html file

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,4 +8,8 @@ const data = fs.readFileSync('./views/login.html','utf8');
 
 app.get('/', (req, res) => res.send(data))
 
+const data1 = fs.readFileSync('./views/game.html','utf8');
```
Comment (oskarszura) [inline index.js]: You have now the following variables:
- `data`
- `data1`
it's not really descriptive, please find more descriptive names like `loginHTML` / `gameHTML` or something similar.

Comment (oskarszura) [reply index.js]: Also move all HTML variables above all `app.get` functions. For the sake of code readibility.

File: views/game.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<html>
```
Comment (oskarszura) [inline views/game.html]: Please cleanup all the tags to have proper indents and remove unnecessary new spaces. Please do it also with the `login.html`.

File: index.js [hunk 3]
```diff
@@ -8,4 +8,8 @@ const data = fs.readFileSync('./views/login.html','utf8');
```
Comment (oskarszura) [inline index.js]: Please move `app.use('/static', express.static('public'));` below template variables. We want to keep loaded variables in one place and Express configuration in another group / block.

**Comment by oskarszura** [comment]
IMPORTANT: This time, please DON'T MERGE IT before I'll let you know, how to do it. We gonna use this time 'Merge & Squash' method :).

**Comment by Birek100** [comment]
I think everything is fixed

**Pull request merged**